src/js/Attachment/index.ts

```diff
@@ -74,6 +72,17 @@ class Attachment extends Control {
             this.listFileEl.appendChild(itemFile.render());
         });
     }
+
+    if (changedAttr.indexOf('isErrorVisible') !== -1) {
+        this.fileErrorEl.style.display = 'none';
+        if (this._props.isErrorVisible === true) {
```

Comment:
Use (this._props.isErrorVisible) instead of (this._props.isErrorVisible === true)